api/graphite_req.go

```diff
@@ -0,0 +1,160 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/grafana/metrictank/api/models"
+)
+
+// ReqMap is a map of requests of data,
+// it has single requests for which no pre-normalization effort will be performed, and
+// requests are that can be pre-normalized together to the same resolution, bundled by their PNGroup
+type ReqMap struct {
+	single   []models.Req
+	pngroups map[models.PNGroup][]models.Req
+	cnt      uint32
+}
+
```

Comment:
```suggestion
// NewReqMap returns a pointer to a new ReqMap with pngroups instantiated
```

Reply:
i really don't think comments with this level of detail are useful. Comments are not meant to describe in detail what the code does. This is a constructor, if the constructor doesn't properly initialize the fields, it is a bug.